tensorrt_llm/_torch/distributed/_ulysses_gc.py

```
@@ -0,0 +1,297 @@
+            self.gc_selfcopy_handle = int(self.gc_selfcopy_stream.cuda_stream)
+        else:
+            try:
+                gc_small, gc_large, s_small, s_large = _create_two_partitions(
+                    device_id, large_sms=large_sms)
+                self._gc_small = gc_small
+                self._gc_large = gc_large
+                # gc_comp on large partition (compute), selfcopy on small.
+                self.gc_comp_stream = s_large
+                self.gc_selfcopy_stream = s_small
+                self.gc_selfcopy_handle = int(s_small.cuda_stream)
+            except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)